README.md

```diff
@@ -518,6 +518,26 @@ User.find_each do |user|
 end
 ```
 
+### Recreating versions with `:from_version` dependency
```

Comment:
Please move this to the wiki, the Readme is already too big for a very specific user case